docs/howto/bootstrap/collect.md

```diff
@@ -43,4 +43,87 @@ flowchart LR
      They also write a script that parses the data from the threat feeds and applies the data map to assign a value to 
      the _Exploitation_ decision point.
 
-{% include-markdown '../evidence_gathering.md' heading-offset=1 %}
```

Comment:
As I was fixing links, I also took advantage of some opportunities to eliminate the need for some file includes by just merging the included content into the parent file.